### PR TITLE
[14.0][OU-IMP] website: convert all HTML fields from version 13 to 14

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
Convert all HTML strings in all HTML fields from Odoo version 13 to the new definitions in version 14.

cc @Tecnativa TT41471

@pedrobaeza @chienandalu please review :)

### CODE REVIEW

**Review by legalsylvain — COMMENTED** [review]
why do you put this call here. (website / end-migration.py). AfAIU, it can be interesting also if website is not installed.

**Comment by pedrobaeza** [comment]
If `website` not installed, then you don't drag and drop blocks (snippets) that contain the HTML that is changing between versions, so outside `website`, it's useless.

**Review by legalsylvain — APPROVED** [review]
> If website not installed, then you don't drag and drop blocks (snippets) that contain the HTML that is changing between versions, so outside website, it's useless.

thanks for the explanation.

File: openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def convert_field_html_string(env):
+    fields = env["ir.model.fields"].search(
+        [("ttype", "=", "html"), ("store", "=", True)]
+    )
+    for field in fields:
+        model = field.model_id.model
+        if env.get(model):
+            if openupgrade.table_exists(env[model]._table):
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py]: Missing `env.cr` parameter, isn't it?

Comment (pilarvargas-tecnativa) [reply openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py]: Yes, sorry, that's it

File: openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+from openupgradelib import openupgrade, openupgrade_140
+
+
+def convert_field_html_string(env):
+    fields = env["ir.model.fields"].search(
+        [("ttype", "=", "html"), ("store", "=", True)]
+    )
+    for field in fields:
+        model = field.model_id.model
+        if env.get(model):
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py]: Sorry, I told you this incorrectly. It should be:

```suggestion
        if env.get(model, False) is not False:
```

Comment (pilarvargas-tecnativa) [reply openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py]: done!

File: openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+from openupgradelib import openupgrade, openupgrade_140
+
+
+def convert_field_html_string(env):
+    fields = env["ir.model.fields"].search(
+        [("ttype", "=", "html"), ("store", "=", True)]
+    )
+    for field in fields:
+        model = field.model_id.model
+        if env.get(model, False) is not False:
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py]:
```suggestion
        if env.get(model, False) is not False and env[model]._auto:
```

Comment (chienandalu) [reply openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py]: Oh, good one :)

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**